### ISSUE TITLE
DKT atlas in BrainColor

### ISSUE THREAD

**Issue by cMadan** [binarybottle/brainCOLORmap#1]
Are there any plans to implement the DKT atlas (31 regions) in BrainColor?

**Comment by binarybottle** [comment]
If you mean constructing an optimal colormap for the DKT labeling protocol, then yes!  I am helping to build a browser-based visualization tool for brain surface data.  I would like the color mapping to provide distinct colors, so for the next prototype I will try to optimize the colormap.

**Comment by cMadan** [comment]
Yes, that's exactly what I meant! It would be great if you could post a note here when that's available.

**Comment by binarybottle** [comment]
Will do!
